ci/upgrade/jiva/jiva-operator.tmp.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright © 2020 The OpenEBS Authors
```

Comment:
Why is it named `*.tmp.yaml` ?

Reply:
The file has some placeholders for the openebs.io/version labels and the images which get replaced in actions. Those are place holders as it depends on the branch like v2.6.x-ci etc.